mediawiki/LocalSettings.php

```diff
@@ -232,8 +247,8 @@
 # $ceAllowConfirmedEmail = true;
 
 # Rate limit to prevent brute-forcing captchas
-# 3 wrong captchas allowed every 10 minutes per IP
-$wgRateLimits['badcaptcha']['ip'] = array( 3, 10 * 60 );
+# 20 wrong captchas allowed every 10 minutes per IP
+$wgRateLimits['badcaptcha']['ip'] = array( 20, 10 * 60 );
```

Comment:
The rate limit might have been the reason the previous digit selection captcha was getting rejected sometimes, so increase it.